src/main/java/appeng/parts/CableBusContainer.java

```diff
@@ -292,11 +294,13 @@ else if( bp != null && !( bp instanceof IPartCable ) && side != AEPartLocation.I
 						{
 							try
 							{
-								new GridConnection( cn, sn, AEPartLocation.INTERNAL );
+								GridConnection.create( cn, sn, AEPartLocation.INTERNAL );
 							}
 							catch( final FailedConnectionException e )
 							{
 								// ekk!
+								e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1148)